install_dependencies.sh

```diff
@@ -36,7 +36,29 @@ echo "Installing Misc. Utilities"
 echo "================================================================"
 
 sudo apt-get install -y\
-    clang-format
+    clang-format\
+    python-rosinstall
+
+
+echo "================================================================"
+echo "Installing Project Dependent ROS packages."
+echo "================================================================"
+
+# Setup rosinstall
+sudo mkdir -p /usr/share/ros/
+sudo chmod a+rwx /usr/share/ros
+rosinstall .
+echo "bash $CURR_DIR/setup.sh" >> ~/.bashrc
+
+echo "================================================================"
+echo "Installing Udev rules for phidgets"
+echo "================================================================"
+
+# Setup udev rules
+sudo cp /usr/share/ros/phidgets_api/share/udev/99-phidgets.rules /etc/udev/rules.d
+echo "Phidgets udev rules have been copied to /etc/udev/rules.d"
+# The reason the script isn't used is because stupid bash/sh doesn't know which directory it's in
+# The actual script that comes with phidget is under /usr/share/ros/phidgets_api/share/setup-udev.sh
```

Comment:
Got a good laugh out of this one. Perhaps we could submit a pull request, or at least open an issue on the upstream repo? Ideally they'll fix that and we can eventually remove this section.